forest/daemon/src/main.rs

```diff
@@ -92,6 +93,13 @@ fn build_daemon<'a>(config: &DaemonConfig) -> anyhow::Result<Daemon<'a>> {
     Ok(daemon)
 }
 
+// Protect user from low FD for its own good
+fn protect_from_low_fd() {
+    if let Some(curr) = raise_fd_limit() {
+        info!("Open-file limit raised to {curr}");
+    }
+}
+
```

Comment:
So if I understand correctly, this will **always** be logged. There's even no point in doing `if let` as this method never returns an `Err` https://docs.rs/fdlimit/latest/src/fdlimit/lib.rs.html#88-110. 

Maybe let's do this only if e.g. soft limit is smaller/equal to rocksdb config?

Reply:
On non-unix OS (albeit we don't support Windows), `None` is returned.

Reply:
Will try to find some heuristic on how much we need FDs and raise only the limit if we can't satisfy it.